proto/workload.proto

```diff
@@ -45,7 +95,12 @@ message Workload {
   string service_account = 7;
 
   // If present, the waypoint proxy for this workload.
-  repeated bytes waypoint_addresses = 8;
+  // All incoming requests must go through the waypoint.
+  GatewayAddress waypoint = 8;
+
+  // If present, East West network gateway this workload can be reached through.
+  // Requests from remote networks should traverse this gateway.
+  GatewayAddress network_gateway = 19;
```

Comment:
how can present multi network gateways

Reply:
this will point to a service which itself can point to workloads in many networks

Reply:
the api has been updated, please take a look again